scripts/o3de/ExportScripts/export_source_built_project.py

```diff
@@ -0,0 +1,252 @@
+
+    :param ctx:                                     The O3DE Script context provided by the export-command
+    :param selected_platform:                       The asset platform to use for the packaging of assets.
+    :param output_path:                             The base output path of the exported package(s)
+    :param should_build_tools:                      Option to build the export process dependent tools (AssetProcessor, AssetBundlerBatch, and dependencies)
+    :param build_config:                            The build configuration to use for the export package launchers
+    :param seedlist_paths:                          List of seed list files to optionally include in the asset bundling process
+    :param game_project_file_patterns_to_copy:      List of game (or unified) launcher specific files to include in the game/unified package
+    :param server_project_file_patterns_to_copy:    List of server (or unified) launcher specific files to include in the server/unified package
+    :param project_file_patterns_to_copy:           List of general file patterns to include in all packages
+    :param preferred_asset_list_path:               Desired location of the generated asset list files
+    :param max_bundle_size:                         Maximum size to set for the archived bundle files
+    :param should_build_all_assets:                 Option to build/process all the assets for the game
+    :param should_build_game_launcher:              Option to build the game launcher package
+    :param should_build_server_launcher:            Option to build the server launcher package
+    :param should_build_unified_launcher:           Option to build the unified launcher package
+    :param allow_registry_overrides:                Option to allow registry overrides in the build process
+    :param tools_build_path:                        Optional build path to build the tools. (Will default to build/tools if not supplied)
+    :param game_build_path:                         Optional build path to build the game launcher(s). (Will default to build/game if not supplied)
+    :param archive_output_format:                   Optional archive format to use for archiving the final package(s)
+    :param fail_on_asset_errors:                    Option to fail the process on any asset processing error
+    :param logger:                                  Optional logger to use to log the process and errors
+    """
+
+    # Use a provided logger or get the current system one
+    if not logger:
+        logger = logging.getLogger()
+        logger.setLevel(logging.ERROR)
+
+    # Calculate the tools and game build paths
+    default_base_build_path = ctx.engine_path / 'build' if ctx.is_engine_centric else ctx.project_path / 'build'
+    if not tools_build_path:
+        tools_build_path = default_base_build_path / 'tools'
+    if not game_build_path:
+        game_build_path = default_base_build_path / 'game'
```

Comment:
I would suggest using the folder and variable name of 'launcher' instead of 'game'.

Using 'game' here means to configure a CMake project folder that can build ALL launchers.
While the other use of  "game" in this function such as "should_build_game_launcher" argument refers specifically to the GameLauncher.